extension.Rmd

```diff
@@ -0,0 +1,241 @@
+
+library(arm) # To create binned plots for logistic models
+binnedplot(predict(model_1), resid(model_1), cex.pts=1, col.int="black")
+
+data_model1 <- data.frame(male = 1, age_1 = 1, age_2 = 0, age_4 = 0, edu_1 = 1, edu_2 = 0, farmer_own = 0, worker = 0, retired = 0, unemployed = 1, partner = 0, left_right_scale = max(data$left_right_scale, na.rm = T), country = "AT")
+
+predict(model_1, data_model1, type = "response")
+```
+
+For model 2, we again plot residuals vs. fitted values. This time the predictions are clustered on the left side. We then extract the predicted probability for an individual who disapproves of EU and is not satisfied with democracy.
+
+```{r}
+
+plot(fitted(model_2), residuals(model_2),
+  xlab = "Fitted Values", ylab = "Residuals")
+  abline(h=0, lty=2)
+
+binnedplot(predict(model_2), resid(model_2), cex.pts=1, col.int="black")
+
+data_model2 <- data.frame(eu_neg = max(data$eu_neg, na.rm = T), z_dem_satis = max(data$z_dem_satis, na.rm = T), country = "AT") 
+
+predict(model_2, data_model2, type = "response")
+```
+
+
+For model 3, after plotting fitted values vs. residuals, we look for predicted probability for an individual whose context of unemployment rates, unemployment benefits, number of asylum seekers, and salience and variance of right-wing issues is Austria's most extreme--most likely to vote for an extreme right party, according to the theory.
+
+```{r}
+plot(fitted(model_3), residuals(model_3),
+  xlab = "Fitted Values", ylab = "Residuals")
+  abline(h=0, lty=2)
+
+binnedplot(predict(model_3), resid(model_3), cex.pts=1, col.int="black")
+
+#max_z_asylumseekers <- max(data$z_asylumseeker, na.rm = T) #4.45935
+#max_z_unemp <- max(data$z_unemp, na.rm = T) #12.2919
+#max_z_asylumseeker_z_unemp <- max(data$z_asylumseeker:data$z_unemp, na.rm = T) #-0.2445196
+#max_z_replacement <- max(data$z_replacement, na.rm = T) #32.96374
+#max_z_replacement_z_unemp <- max(data$z_replacement:data$z_unemp, na.rm = T) #0.5738534
+#max_z_replacement_z_asylumseeker <- max(data$z_replacement:data$z_asylumseeker, na.rm = T) #0.5738534
+
+#data_model3 <- data.frame(max_z_asylumseekers, max_z_unemp, max_z_asylumseeker_z_unemp, max_z_replacement, max_z_replacement_z_unemp, max_z_replacement_z_asylumseeker, country = "AT")
+#predict(model_3, data_model3, type = "response")
+#We have thus far been unable to debug the above code such that we could generate a prediction.  However, we do have a binned residual plot for this model.  
+```
+
+The binned residual plots show the highest level of randomness for the second and third models, Parsons' and the group conflict theories.  This method would indicate that, contrary to our conclusions from the separation plots (see below), these models have the closest fit to the observed data.  However, all of these conclusions must be qualified:  the proxy variables available to us in this data set do not perfectly operationalize the theoretical models in question.
```

Comment:
More generally I did not understand these diagnostics, perhaps because I have not worked with logistic regression much myself